### PR TITLE
[sp] update chart styling

### PR DESCRIPTION
**Pull request by shrey-mage**
Marking as draft for now since I'm not sure which other charts we'd want to restyle. This is an update on what's ready to ship.

# Summary
- update horizontal bar chart to match Figma design styling
- update pie chart to use lighter colors
- add new design colors and update existing ones to match Figma

# Tests
image
image

cc: @johnson-mage @dy46

### CODE REVIEW

**Comment by johnson-mage** [comment]
Please make the left side of the horizontal bar radius match the right side. Seems too rounded on the left right now. @shrey-mage

File: mage_ai/frontend/components/charts/BarGraphHorizontal.tsx [hunk 1]
```diff
@@ -208,20 +208,29 @@ const BarChartHorizontal = withTooltip<BarStackHorizontalProps, TooltipData>(
...
-                      />
-                      <rect
-                        fill="url(#bar-linear-gradient)"
-                        height={bar.height}
-                        pointerEvents="none"
-                        width={bar.width}
-                        x={bar.x}
-                        y={bar.y}
-                      />
+                      <>
```
Comment (dy46) [inline mage_ai/frontend/components/charts/BarGraphHorizontal.tsx:211]: nit: why do you need this empty fragment?

**Comment by dy46** [comment]
is the `country` graph in the screenshot correct? it looks like it's cut off or something to me

File: mage_ai/frontend/components/charts/constants.ts [hunk 2]
```diff
@@ -1,50 +1,26 @@
...
-  water300,
-  fire300,
-  energy300,
-  earth300,
-  wind200,
-  water200,
-  fire200,
-  energy200,
-  earth200,
+  light.brand.wind400,
```
Comment (johnson-mage) [inline mage_ai/frontend/components/charts/constants.ts:4]: @shrey-mage we prob cud'be kept the object destructuring.

**Review by johnson-mage — APPROVED** [review]

**Pull request merged**